src/test/resources/application.conf

```diff
@@ -12,16 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 "monitoringCluster": "mad_test",
-"logDirectory": "./logs",
-"akka" {
-  "actor" {
-    "provider" = "akka.remote.RemoteActorRefProvider"
```

Comment:
So I was under the impression that this particular configuration was required to test the Actor module? -- I think it's been removed from MAD (now only in CAgg -- and soon to be commons) but this makes my point above about different actor system configurations per test. I would say as a starting point that would be a requirement. Ideally, there can be a base config with the log and cluster as here assuming you can reference and include it from each test specific one.